### PR TITLE
Add extra test case for resolving browser aliases in subdirectories

### PR DESCRIPTION
**Pull request by DeMoorJasper**
---
name: 🙋 New Pull Request
about: Do you want to change something on parcel?
---

## ↪️ Pull Request

A test case to try and recreate #2054

## ✔️ PR Todo

- [x] Added/updated unit tests for this change
- [x] Filled out test instructions
- [x] Included links to related issues/PRs

### CODE REVIEW

**Comment by devongovett** [comment]
@DeMoorJasper this test appears to be failing on Windows. Looks like it's using the node version instead of the client version there. Perhaps that's the root of the original bug reported by @thsorens?

**Comment by thsorens** [comment]
> @DeMoorJasper this test appears to be failing on Windows. Looks like it's using the node version instead of the client version there. Perhaps that's the root of the original bug reported by @thsorens?

Yes, this was when running Windows

File: src/Resolver.js [hunk 1]
```diff
@@ -392,6 +392,11 @@ class Resolver {
   }
 
   lookupAlias(aliases, filename, dir) {
+    filename = path.normalize(filename);
+    Object.keys(aliases).forEach(
+      key => (aliases[path.normalize(key)] = aliases[key])
+    );
```
Comment (devongovett) [inline src/Resolver.js]: Probably not a good idea to mutate the package.json... Maybe do the normalization + checking in the existing for loop below instead

Comment (DeMoorJasper) [reply src/Resolver.js]: Probably a better idea, will change it

**Comment by devongovett** [comment]
Due to move to monorepo which rewrote the git history, this PR was automatically closed. Please open a new PR against master with your changes. You can do a diff against the `1.x` branch to see them. Sorry about that!

**Pull request closed without merging**